### PR TITLE
ESLint for Bassa UI

### PR DESCRIPTION
**Pull request by kmehant**

### CODE REVIEW

**Comment by kmehant** [comment]
@agentmilindu

File: ui/.eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+extends: airbnb-base
+plugins:
+    - import
+rules:
+    no-underscore-dangle: [2, {allowAfterThis: true}]
+    linebreak-style: [error, windows]
```
Comment (agentmilindu) [inline ui/.eslintrc.yml]: I'm not sure if this `linebreak-style: windows ` would give Linux users troubles.

Comment (kmehant) [reply ui/.eslintrc.yml]: shall i remove this rule?

Comment (agentmilindu) [reply ui/.eslintrc.yml]: @rehrumesh @malithsen WDYT? I'm not very familiar with ESLint

**Pull request merged**